service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/hearing/NewNoticeOfHearingTemplate.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.fpl.model.notify.hearing;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import uk.gov.hmcts.reform.fpl.model.notify.NotifyData;
+
+import java.util.Map;
+
+@Getter
+@Setter
+@Builder
```

Comment:
Do we need both setter and builder? It doesn't matter, I guess, but could we use Data?